ci-bootstrap/terraform/role.tf.tmpl

```diff
@@ -0,0 +1,87 @@
+# Run via ci-bootstrap/bin/deploy.sh
+
+provider "aws" {
+    region = "eu-west-2"
+}
+
+variable "environment" {}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_iam_role" "deploy-attachment-lambda" {
+  name = "bpm-svc-deploy-attachment-lambda"
+  assume_role_policy = data.aws_iam_policy_document.instance-assume-role-policy.json
+}
+
+resource "aws_iam_role_policy" "deploy-attachment-lambda-inline" {
+  name = "DeployAttachmentLambdaPolicy"
+  role = aws_iam_role.deploy-attachment-lambda.name
+  policy = data.aws_iam_policy_document.bpm-deploy-attachment-lambda-policy.json
+}
+
+data "aws_iam_policy_document" "instance-assume-role-policy" {
+  statement {
+    sid = "AssumeDeployRole"
+    actions = [
+      "sts:AssumeRole"
+    ]
+    principals {
+      type = "AWS"
+      identifiers = [
+        "arn:aws:iam::736089984042:user/bpm-svc-concourse"
+      ]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "bpm-deploy-attachment-lambda-policy" {
+  statement {
+    sid = "InlineIAMLambdaS3Policy"
+    effect = "Allow"
+    actions = [
+                "s3:List*",
```

Comment:
Does it need all these s3 permissions just to deploy the lambda?

Reply:
Yes, I think so.

Reply:
The lambda itself only has two, but this is to create the bucket, there's nothing in the bucket, so what does it need to list or get?

Reply:
I will run a test on that to make sure and will let you know.

Reply:
Doesn't work without list and get. Keeping it